### PR TITLE
feat: Read receipts

### PR DESCRIPTION
**Pull request by JPonte**
#### APK
[Download build #12129](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12129/artifact/build/artifact/wire-dev-PR1894-12129.apk)
[Download build #12130](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12130/artifact/build/artifact/wire-dev-PR1894-12130.apk)
[Download build #12141](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12141/artifact/build/artifact/wire-dev-PR1894-12141.apk)
[Download build #12142](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12142/artifact/build/artifact/wire-dev-PR1894-12142.apk)
[Download build #12143](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12143/artifact/build/artifact/wire-dev-PR1894-12143.apk)
[Download build #12144](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12144/artifact/build/artifact/wire-dev-PR1894-12144.apk)
[Download build #12148](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12148/artifact/build/artifact/wire-dev-PR1894-12148.apk)
[Download build #12149](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12149/artifact/build/artifact/wire-dev-PR1894-12149.apk)
[Download build #12150](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12150/artifact/build/artifact/wire-dev-PR1894-12150.apk)
[Download build #12151](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12151/artifact/build/artifact/wire-dev-PR1894-12151.apk)
[Download build #12152](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12152/artifact/build/artifact/wire-dev-PR1894-12152.apk)
[Download build #12153](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12153/artifact/build/artifact/wire-dev-PR1894-12153.apk)
[Download build #12156](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12156/artifact/build/artifact/wire-dev-PR1894-12156.apk)
[Download build #12158](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12158/artifact/build/artifact/wire-dev-PR1894-12158.apk)
[Download build #12166](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12166/artifact/build/artifact/wire-dev-PR1894-12166.apk)
[Download build #12168](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12168/artifact/build/artifact/wire-dev-PR1894-12168.apk)
[Download build #12177](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12177/artifact/build/artifact/wire-dev-PR1894-12177.apk)
[Download build #12178](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12178/artifact/build/artifact/wire-dev-PR1894-12178.apk)
[Download build #12179](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12179/artifact/build/artifact/wire-dev-PR1894-12179.apk)
[Download build #12181](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12181/artifact/build/artifact/wire-dev-PR1894-12181.apk)
[Download build #12183](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12183/artifact/build/artifact/wire-dev-PR1894-12183.apk)
[Download build #12185](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12185/artifact/build/artifact/wire-dev-PR1894-12185.apk)
[Download build #12186](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12186/artifact/build/artifact/wire-dev-PR1894-12186.apk)

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -21,7 +21,7 @@ ext {
     playServicesVersion = '15.0.1'
     audioVersion = System.getenv("AUDIO_VERSION") ?: '1.209.0@aar'
     stethoVersion = '1.5.0'
-    zMessagingDevVersion = "133.579-PR@aar"
+    zMessagingDevVersion = "133.586-PR"
```
Comment (deanrobertcook) [inline app/build.gradle]: I find it strange that this works...

**Comment by red-avtovo** [comment]
Can't see read receipts for ping in 1:1

**Comment by red-avtovo** [comment]
group conversations are broken for Pro users:
- unable to open a group after creation
- after restarting the app unable to sent the message (message shows as Sent, but it also still present in input field)

**Comment by red-avtovo** [comment]
Some bugs from cross-platform test

Added to the group
==================
- group icon is like 1:1 with myself (happened with group, where read receipt is already ON) => From linear group creation
- no system message about current read receipt state (works from linear group creation, but not with existing groups)
- no read status on pings

In group
========
Sent a message and edited that after first read
===============================================
- After editing, I receive second read receipt from the same person and even show it in the list

Read receipt is off for group
=============================
- Wrong copy read placeholder in message details view

Design: System message padding is weird between image and message

Ephemeral messages
==================
- No read receipt status for sent ephemeral message
- LIKED tab for sent ephemeral message

**Comment by red-avtovo** [comment]
New team user should have Read Receipts ON by default, but right now it is OFF

**Comment by red-avtovo** [comment]
Popup shows up after first switch of Read Receipts toggle from `null` to `ON/OFF`

**Comment by red-avtovo** [comment]
It is possible to like self pings

**Pull request merged**